koku/api/cloud_accounts/views.py

```diff
@@ -15,23 +15,49 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 """View for Cloud Account."""
-from rest_framework import viewsets
-from rest_framework.permissions import AllowAny
+import json
+import logging
+import os
 
-from api.cloud_accounts.models import CloudAccount
-from api.cloud_accounts.serializers import CloudAccountSerializer
+from django.core.paginator import Paginator
+from django.shortcuts import render
+from rest_framework import permissions
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.decorators import permission_classes
+from rest_framework.decorators import renderer_classes
+from rest_framework.renderers import BrowsableAPIRenderer
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
 
+from koku.settings import STATIC_ROOT
 
-class CloudAccountViewSet(viewsets.ReadOnlyModelViewSet):
-    """View for Cloud Accounts."""
+LOG = logging.getLogger(__name__)
 
-    serializer_class = CloudAccountSerializer
-    permission_classes = (AllowAny,)
+OPENAPI_FILE_NAME = os.path.join(STATIC_ROOT, "cloud_accounts.json")
+"""View for Cloud Accounts."""
 
-    def get_queryset(self):
-        """Override default get_queryset to filter on name."""
-        queryset = CloudAccount.objects.all()
-        cloud_account = self.request.query_params.get("name", None)
-        if cloud_account is not None:
-            queryset = queryset.filter(name=cloud_account)
-        return queryset
+
+def get_json(path):
```

Comment:
You should just store this as python not JSON.  The first thing you did is load it into python anyway.  This will also help with query params.

Reply:
@kholdaway - This is following the same pattern here that we already use for serving up the `openapi.json`. 

The cloud account info needs to be stored in a human-editable format so that we can change it when needed. This data should not be pickled or stored in any other non-editable format.

Reply:
@kholdaway where do you want this Python object to live?

Reply:
@kholdaway I plan to store this Python object in /cloud_accounts/cloud_accounts_data.py 
Let me know if you think the location should change.

Reply:
@blentz There is a query parameter so I would think we may need to filter.  Is that incorrect?  Also, the code did a json.load immediate (python again anyway)

Reply:
see @blentz https://github.com/project-koku/koku/pull/1938/files#diff-60437d138c668df7d7e3346cc6fd97c4R65-R72

Reply:
@kholdaway - To me, this is about mantaining separation of concerns. This contents of the json is data, not code. So, storing it as json is an appropriate solution and allows us to change it on the fly (e.g. with an openshift mounted secret file).

Hard-coding this information into the code means that it's less flexible and can't be changed without going through the entire release process. 

My suggestion to Nick to go this route is based on my years of experience operating webservices at scale and knowing these are the kinds of knobs that it's good to leave exposed to maintain service flexibility. During my time in OpenShift Operations, we moved AWS accounts more than once. So, I'm just trying to get us a solution that we won't have to change later.

However, I don't feel the need to argue the point. I worked with Nick yesterday to remove the json file and just use a hard-coded version as you and Chris were requesting. So, the point is moot. As soon as he pushes his latest changes, you'll have the implementation you were asking for.

Reply:
@blentz Do we have a plan to co-located all configuration files?  @nbon12 is going to do this for several more similar issues.  If there is a defined configuration area then I think I could agree to the separation of concerns.  We should make sure the documentation covers that configuration.